core/repository/manager/src/main/java/org/eclipse/rdf4j/repository/manager/RemoteRepositoryManager.java

```diff
@@ -233,7 +196,7 @@ public RepositoryConfig getRepositoryConfig(String id) throws RepositoryExceptio
 	}
 
 	@Override
-	public Collection<RepositoryInfo> getAllRepositoryInfos(boolean skipSystemRepo) throws RepositoryException {
```

Comment:
This isn't backwards compatible and the method wasn't deprecated. Did you introduce a default interface method that is deprecated and maybe logs an error or just simply ignores the provided param?